composer.json

```diff
@@ -37,6 +37,7 @@
         "laravel/tinker": "^2.8",
         "league/flysystem-aws-s3-v3": "^3.3",
         "livewire/livewire": "^3.0",
+        "livewire/volt": "^1.6",
```

Comment:
Probably should've installed this when we began the Folio transition. Volt enables Livewire to play nicely with Folio and use a Folio-like functional syntax. [docs](https://livewire.laravel.com/docs/volt)